tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx

```diff
@@ -124,7 +124,7 @@ void ROOT::Experimental::RNTupleDescriptor::PrintInfo(std::ostream &output) cons
 
          info.fNElements += columnRange.fNElements;
          if (compression == -1) {
-            compression = columnRange.fCompressionSettings;
+            compression = columnRange.fCompressionSettings.value();
```

Comment:
I think we should not throw here if the compression settings is unknown: even if we don't expect it to happen, this is a method used for informative reasons and it can handle the unexpected case in a clear manner (print "unknown"), so I think it should.